lib/pka.c

```diff
@@ -583,6 +587,11 @@ static int pka_cmd_enqueue(pka_global_info_t    *gbl_info,
     alloc.ring   = ring_info;
     // Allocate some window RAM for the total set vectors.
     base_offset  = pka_mem_alloc(ring_info->ring_id, cmd_desc->operands_len);
+    if (0 == base_offset )
+    {
+        PKA_DEBUG(PKA_USER, "there are no rings available\n");

```

Comment:
Apt error message is "No memory available in the ring" ?
Also, if there is no memory available within a ring, does it make sense to try and use a different ring (if available) before returning error ?

Reply:
This happens during bug fixing. Probably would never happen in normal situation. It shall indicate where the error is if this happens in the future.